docs/topics/forms/modelforms.txt

```diff
@@ -475,9 +475,8 @@ Overriding the default fields
 
 The default field types, as described in the `Field types`_ table above, are
 sensible defaults. If you have a ``DateField`` in your model, chances are you'd
-want that to be represented as a ``DateField`` in your form. But
-``ModelForm`` gives you the flexibility of changing the form field type and
-widget for a given model field.
```

Comment:
I didn't think it was that helpful to single out widgets here, "changing the form field for a given model" includes customising the widget. The example for overriding widget starts two lines below, then followed by all the other attributes that can be overridden in the same way.

You want it back?